client/send-text.cc

```diff
@@ -175,6 +185,24 @@ int main(int argc, char *argv[]) {
             }
         } while (run_forever && !interrupt_received);
     }
+    else if ( rot_vertical == true) { 
```

Comment:
* Here, you are not checking for scroll_delay_ms > 0. You might do that as well by just having one
big if around
```
    if (scroll_delay_ms > 0) {
          if (!rot_vertical) {
                // regular 
          } else {
                // vertical
          }
    }
    else {
         // don't scroll
    }
```
 * There is a superfluous space after the open parenthesis.
 * It is more readable to use booleans directly in expressions instead of comparing them with constants. So `if (foo)` instead of `if (foo == true)`  or `if (!foo)` instead of `if (foo == false)`